### PR TITLE
Add the MFTF Credentials topic

### PR DESCRIPTION
**Pull request by dshevtsov**
## This PR is a:

New topic

## Summary

When this pull request is merged, it will add a topic describing usage of sensitive data in the MFTF.

whatsnew
Added a topic about [Credentials in the MFTF](https://devdocs.magento.com/mftf/2.3/credentials.html).

### CODE REVIEW

**Comment by dshevtsov** [comment]
@KevinBKozan I think it makes sense to cover the use of credentials in Magento PRs when we'll have the corresponding solution.
We can add a note that the out-of-the-box MFTF tests delivered with Magento does not use credentials and does not cover external services, because of the data sensitivity.

File: mftf/2.3/credentials.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+# Credentials for the MyAwesome service
+my_awesome_service_token=rRVSVnh3cbDsVG39oTMz4A
+
+# Credentials for the USPS service
+carriers_usps_userid=test_user
+carriers_usps_password=Lmgxvrq89uPwECeV
+....
+```
+
+## Use credentials in test
```
Comment (keharper) [inline mftf/2.3/credentials.md]: "test" seems not quite right. Maybe change to "a test", "tests", or "testing"?

File: mftf/2.3/credentials.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- `_CREDS` is an environment constant pointing to the `.credentials` file
+- `my_data_key` is a key in the the `.credentials` file that contains the value to be used in a test step
+
+For example:
+
+```xml
+<fillField stepKey="FillApiToken" selector=".api-token" userInput="{{_CREDS.my_data_key}}" />
+```
+
+## Implementations details
```
Comment (keharper) [inline mftf/2.3/credentials.md]:
```suggestion
## Implementation details
```

File: mftf/2.3/credentials.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+The generated tests does not contain credentials values.
```
Comment (keharper) [inline mftf/2.3/credentials.md]:
```suggestion
The generated tests do not contain credentials values.
```

**Pull request merged**